arch/arc/core/fault_s.S

```diff
@@ -36,6 +36,15 @@ GTEXT(__ev_maligned)
 
 GDATA(exc_nest_count)
 
+	.balign 4
+SECTION_VAR(BSS, saved_value)
+	.word 0
+
+#define EXCEPTION_STACK_SIZE 256
```

Comment:
instead of doing this, why not use the interrupt stack for exceptions like all our other arches?